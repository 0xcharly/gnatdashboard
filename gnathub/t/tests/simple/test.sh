# launch GNAThub on a trivial project
gnathub -q -P default --plugins gnatmetric,gnatcheck,sonarconfig

# verify that the object directory contains the expected files
# at the expected locations
find obj/ | grep -v auto.cgpr |  sort
