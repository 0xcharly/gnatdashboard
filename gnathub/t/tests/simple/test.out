obj/
obj/empty
obj/gnatcheck.out
obj/gnatcheck-source-list.out
obj/gnathub
obj/gnathub/gnathub.db
obj/gnathub/logs
obj/gnathub/logs/GNATcheck.log
obj/gnathub/logs/GNATmetric.log
obj/gnathub/sonar
obj/gnathub/sonar/sonar-project.properties
obj/hello.adb.metrix
obj/metrix.xml
